docker/quarto/Dockerfile

```diff
@@ -0,0 +1,36 @@
+ARG VAST_VERSION
+ARG QUARTO_VERSION
+
+FROM tenzir/vast:$VAST_VERSION
+ARG QUARTO_VERSION
+ARG TARGETPLATFORM
+ARG TARGETARCH
+
+USER root
+
+RUN apt-get update && \
+    apt-get install -y \
+        r-base \
+        python3-pip \
+        git \
+        curl && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN apt-get update && \
+    apt-get install -y python3-pip curl && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN git clone -b v${QUARTO_VERSION} https://github.com/quarto-dev/quarto-cli.git /opt/quarto && \
+    cd /opt/quarto && \
+    ./configure.sh
+
+USER vast:vast
+
+# We install Poetry alongside VAST, they shouldn't disturb each other :)
+ENV POETRY_HOME=$PREFIX
+
+RUN curl -sSL https://install.python-poetry.org | python3 -
```

Comment:
I also installed poetry in the upstream image as part of #2636. We need to make sure we don't run into conflicts here.

Reply:
I'll wait for your thing to be merged, then I'll rebase on it and we'll see what happens 😅. Leaving this conversation open as a reminder!

Reply:
I just realized, the poetry you install in https://github.com/tenzir/vast/pull/2636 only lives in the `dependencies` and `development` stages, so we should be fine here as we start of `production` 😉

Reply:
I don't think that's the case, but if you merge this one before #2636 please just leave a comment on that PR to double-check to shift the responsibility.